apache-mesos-mixin/dashboards/apache-mesos-overview.libsonnet

```
@@ -1317,10 +1319,22 @@ local agentLogsPanel = {
               allValues='.+',
               sort=1
             ),
+            template.new(
+              'cluster',
+              promDatasource,
+              'label_values(mesos_exporter_build_info{%(multiclusterSelector)s}, cluster)' % $._config,
+              label='Cluster',
+              refresh=2,
+              includeAll=true,
+              multi=true,
+              allValues='.*',
+              hide=if $._config.enableMultiCluster then '' else 'variable',
+              sort=0
+            ),
             template.new(
               'instance',
               promDatasource,
-              'label_values(mesos_exporter_build_info{job=~"$job", instance=~"$instance"}, instance)',
+              'label_values(mesos_exporter_build_info{%(mssqlSelector)s}, instance)' % $._config,
```

```suggestion
              'label_values(mesos_exporter_build_info{%(mesosSelector)s}, instance)' % $._config,
```
Likely copy-paste error, same thing happened on the `mesos_cluster` selector